warehouse/templates/pages/help.html

```diff
@@ -639,9 +640,10 @@ <h3 id="file-size-limit">{{ file_size_limit() }}</h3>
 
         <h3 id="project-size-limit">{{ project_size_limit() }}</h3>
         <p>
+          {% set max_project_size_gb = request.registry.settings.get('warehouse.forklift.legacy.MAX_PROJECT_SIZE_GIB') %}
           {% trans %}
             If you can't upload your project's release to PyPI because you're
-            hitting the project size limit, first remove any unnecessary
+            hitting the project size limit ({{ max_project_size_gb }} GiB; individual projects may differ), first remove any unnecessary
```

Comment:
```suggestion
            hitting the project size limit ({{ max_project_size_gb }} GiB by default; individual projects may differ), first remove any unnecessary
```

Reply:
Thanks for fixing this!  I meant to add that to both places, but missed the second one.